src/common/function/src/handlers.rs

```diff
@@ -30,6 +31,24 @@ pub trait TableMutationHandler: Send + Sync {
 
     /// Delete rows from the table.
     async fn delete(&self, request: DeleteRequest, ctx: QueryContextRef) -> Result<AffectedRows>;
+
+    /// Migrate a region from source peer to target peer, returns the procedure id if success.
```

Comment:
The doc here is a bit of ambiguity here, whether the procedure id is returned after the entire region migrate is completed, Or the procedure id is returned after the region migrate task is successfully submitted.

Reply:
All procedures are processed asynchronously,  so it will return the procedure id after the task is submitted. The task state can be queried by the `query_state` rpc.